model/shared/src/main/scala/hmda/model/publication/reports/ReportTypeEnum.scala

```diff
@@ -0,0 +1,14 @@
+package hmda.model.publication.reports
+
+import enumeratum.{ Enum, EnumEntry }
+
+sealed abstract class ReportTypeEnum extends EnumEntry
+
+object ReportTypeEnum extends Enum[ReportTypeEnum] {
+
+  val values = findValues
+
+  case object Disclosure extends ReportTypeEnum
+  case object Aggregate extends ReportTypeEnum
+  case object Nationwide extends ReportTypeEnum
```

Comment:
Recommend updating "Nationwide" to "National Aggregate" as there is a separate filter by nationwide.